content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -644,7 +644,7 @@ in Go files or in the
 | Golang marker | OpenAPI extension | Accepted values | Description | Introduced in |
 |---|---|---|---|---|
 | `//+listType` | `x-kubernetes-list-type` | `atomic`/`set`/`map` | Applicable to lists. `atomic` and `set` apply to lists with scalar elements only. `map` applies to lists of nested types only. If configured as `atomic`, the entire list is replaced during merge; a single manager manages the list as a whole at any one time. If `granular`, different managers can manage entries separately. | 1.16          |
-| `//+listMapKeys` | `x-kubernetes-list-map-keys` | Slice of map keys that uniquely identify entries for example `["port", "protocol"]` | Only applicable when `+listType=map`. A slice of strings whose values in combination must uniquely identify list entries. | 1.16 |
+| `//+listMapKey` | `x-kubernetes-list-map-keys` | Slice of map keys that uniquely identify entries for example `["port", "protocol"]` | Only applicable when `+listType=map`. A slice of strings whose values in combination must uniquely identify list entries. | 1.16 |
```

Comment:
Correct, we can have multiple keys (`x-kubernetes-list-map-keys` is plural in the openapi), but in the Go type, we specify them one by one, hence singular ;-)

Reply:
> Correct, we can have multiple keys (`x-kubernetes-list-map-keys` is plural in the openapi), but in the Go type, we specify them one by one, hence singular ;-)

We could probably include that explanation in the description by the way.

Reply:
> We could probably include that explanation in the description by the way.

Updated :)